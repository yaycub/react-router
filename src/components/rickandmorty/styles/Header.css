.Header{
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 8vh;
  background-color: black;
  padding: 10px;
  border-bottom: 5px solid darkgray;

  h1{
    font-size: 20px;
    font-family: Arial, Helvetica, sans-serif;
    color: bisque;
    font-weight: bold;
  }

  nav{
    font-family: Arial, Helvetica, sans-serif;
  }
}

.Link{
  list-style: none;
  color: bisque;
  text-decoration: none;
}

.Link:hover{
  color: darkgray;
}