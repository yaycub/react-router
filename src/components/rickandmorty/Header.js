import React from 'react';
import styles from './styles/Header.css';
import { Link } from 'react-router-dom';

const Header = () => (
  <header className={styles.Header}>
    <h1>React and Morty</h1>
    <nav>
      <li><Link to="/" className={styles.Link}>Home</Link></li>
    </nav>
  </header>
);

export default Header;
