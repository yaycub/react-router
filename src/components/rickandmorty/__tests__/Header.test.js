import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header Component', () => {
  it('should render a Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
