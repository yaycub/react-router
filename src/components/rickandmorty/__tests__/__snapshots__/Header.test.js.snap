// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render a Header 1`] = `
<header
  className="Header"
>
  <h1>
    React and Morty
  </h1>
  <nav>
    <li>
      <Link
        className="Link"
        to="/"
      >
        Home
      </Link>
    </li>
  </nav>
</header>
`;
